### PR TITLE
ref: apps esm, ceramic v2

### PR DESCRIPTION
**Pull request by zachferland**
All builds, remove-postmsg pr broke expected functionality at moment

### CODE REVIEW

**Comment by zachferland** [comment]
Wasn't able to fix png compilation issue yet, but works/builds otherwise.

File: apps/management/next.config.js [hunk 1]
```diff
@@ -6,6 +6,19 @@ const withImages = require('next-images')
 
 const nextConfig = {
   future: { webpack5: true },
```
Comment (PaulLeCam) [inline apps/management/next.config.js]: I think we can remove this, should be the default with newer versions of Next

File: apps/management/next.config.js [hunk 2]
```diff
@@ -6,6 +6,19 @@ const withImages = require('next-images')
...
+  webpack: (config) => {
```
Comment (PaulLeCam) [inline apps/management/next.config.js]: Do we need these? AFAIK Next already has a setup for this

File: apps/management/src/client/components/IdentitiesList.tsx [hunk 3]
```diff
@@ -1,10 +1,10 @@
-import type { Manager } from '@3id/manager'
+import type { Manager } from '@3id/did-manager'
 import { Avatar, Box, Text } from 'grommet'
 import { useMemo } from 'react'
 
 import Button from '../../components/Button'
 import { formatDID, getImageSrc, longFormatDID } from '../../data/idx'
-import avatarPlaceholder from '../../images/avatar-placeholder.png'
+// import avatarPlaceholder from '../../images/avatar-placeholder.png'
```
Comment (PaulLeCam) [inline apps/management/src/client/components/IdentitiesList.tsx]: Next supports importing media like this but returns an object with metadata, so to get the URL you probably need to use `avatarPlaceholder.src`, see https://github.com/ceramicstudio/self.id/blob/main/apps/self.id/src/auth.ts#L10 for example

**Review by PaulLeCam — APPROVED** [review]

File: apps/management/next.config.js (hunk 2 above)
Comment (zachferland) [reply apps/management/next.config.js]: did get errors, but probs not needed if remove above (as mentioned) and use defaults, ill try it

File: apps/management/src/client/components/IdentitiesList.tsx (hunk 3 above)
Comment (zachferland) [reply apps/management/src/client/components/IdentitiesList.tsx]: ah ok 👍

**Comment by zachferland** [comment]
img issues resolved and webpack config removed

**Pull request merged**